doc/stdlib/file.rst

```diff
@@ -40,6 +40,15 @@
       This function raises an error under operating systems that do not support
       soft symbolic links, such as Windows XP.
 
+.. function:: readlink(path) -> AbstractString
+
+   Returns the value of a symbolic link ``path``.
+
+   .. note::
+
+      This function raises an error under operating systems that do not support
```

Comment:
I don't think this note is true, unless `uv_fs_readlink` errors internally on XP.